lib/controller.js

```diff
@@ -181,6 +182,8 @@ Tessel.list = function(opts) {
 };
 
 Tessel.get = function(options) {
+  var discover = require('./discover');
+
```

Comment:
I believe so, yes. When the `discover` module is always required by this file, any CLI action can be blocked by the USB driver error, including things like `t2 init`. Requiring the `discover` only when needed, _should_ lead to a nicer user experience.